internal/provisioner/model/names.go

```diff
@@ -82,17 +82,19 @@ func (c *Contour) EnvoyRBACNames() RBACNames {
 // ComponentLabels returns labels for a Contour component.
 func (c *Contour) ComponentLabels() map[string]string {
 	labels := map[string]string{
-		"app.kubernetes.io/name":       "contour",
-		"app.kubernetes.io/instance":   c.Name,
-		"app.kubernetes.io/component":  "ingress-controller",
-		"app.kubernetes.io/managed-by": "contour-gateway-provisioner",
+		"app.kubernetes.io/instance": c.Name,
 	}
 
 	// Add owner labels
 	for k, v := range OwnerLabels(c) {
 		labels[k] = v
 	}
 
+	// Add user-defined labels
```

Comment:
this should probably come after the contour-defined owner labels so the owner ones override things set by the user, in case they conflict

unless of course we see a compelling reason to let a user override these labels

a unit test for this would be great to add

Reply:
I changed from `componentLabels` to  `commonLabels`, with my use-case, I must filter out the resources which created by myself, not only by `provisioner`,  TBH, I don't know whether  it is a common demand or not

Reply:
i will adjust the codes after you and @skriss  and someone else, think it's a good idea to add this atrribute